src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+            malicious = True
+            attacks = int(str(response).split("attacks: ")[1].split("<")[0])
+            reports = int(str(response).split("reports: ")[1].split("<")[0])
+        try:
+            results = dshield.ip(str(dom["dns_aaaa"][0]), return_format=dshield.JSON)
+            results = json.loads(results)
+            threats = results["ip"]["threatfeeds"]
+            attacks = results["ip"]["attacks"]
+            attacks = int(0 if attacks is None else attacks)
+            malicious = True
+            dshield_attacks = attacks
+            dshield_count = len(threats)
+        except KeyError:
+            dshield_attacks = 0
+            dshield_count = 0
+
+    # Clean-up other fields
+    if "ssdeep_score" not in dom:
+        dom["ssdeep_score"] = ""
+    if "dns_mx" not in dom:
+        dom["dns_mx"] = [""]
+    if "dns_ns" not in dom:
+        dom["dns_ns"] = [""]
+
+    # Ignore duplicates
+    permutation = dom["domain"]
+    if permutation in perm_list:
+        return None, perm_list
+    else:
+        perm_list.append(permutation)
+
+    domain_dict = {
+        "organizations_uid": pe_org_uid,
+        "data_source_uid": source_uid,
+        "sub_domain_uid": sub_domain_uid,
+        "domain_permutation": dom["domain"],
+        "ipv4": dom["dns_a"][0],
+        "ipv6": dom["dns_aaaa"][0],
+        "mail_server": dom["dns_mx"][0],
+        "name_server": dom["dns_ns"][0],
+        "fuzzer": dom["fuzzer"],
+        "date_active": date,
+        "ssdeep_score": dom["ssdeep_score"],
+        "malicious": malicious,
+        "blocklist_attack_count": attacks,
+        "blocklist_report_count": reports,
+        "dshield_record_count": dshield_count,
+        "dshield_attack_count": dshield_attacks,
+    }
+    return domain_dict, perm_list
+
+
+def execute_dnstwist(root_domain, test=0):
+    """Run dnstwist on each root domain."""
+    pathtoDict = str(pathlib.Path(__file__).parent.resolve()) + "/data/common_tlds.dict"
+    dnstwist_result = dnstwist.run(
+        registered=True,
+        tld=pathtoDict,
+        format="json",
+        threads=8,
+        domain=root_domain,
+    )
+    if test == 1:
+        return dnstwist_result
+    finalorglist = dnstwist_result + []
+    for dom in dnstwist_result:
+        if ("tld-swap" not in dom["fuzzer"]) and ("original" not in dom["fuzzer"]):
+            LOGGER.info("Running again on %s", dom["domain"])
+            secondlist = dnstwist.run(
+                registered=True,
+                tld=pathtoDict,
+                format="json",
+                threads=8,
+                domain=dom["domain"],
+            )
+            finalorglist += secondlist
+    return finalorglist
+
+
+def run_dnstwist(orgs_list):
+    """Run DNStwist on certain domains and upload findings to database."""
+    PE_conn = connect()
+    source_uid = getDataSource(PE_conn, "DNSTwist")[0]
+
+    """ Get P&E Orgs """
```

Comment:
Unnecessary capitalization:
```suggestion
    """ Get P&E orgs """
```